lib/urbanopt/geojson/schema/building_properties.json

```diff
@@ -283,6 +283,18 @@
 		"ev_curtailment_frac": {
 			"description": "Fraction between 0 and 1 that denotes curtailment of EV charging load to better align EV charging with expected energy production from a solar PV system",
 			"type": "number"
+		},
+		"characterize_residential_buildings_from_buildstock_csv":{
```

Comment:
Why can't the existence of a populated `resstock_buildstock_csv_path` field serve as the boolean here?